com/metacube/DSA/Assignment4/EmployeeCollection.java

```diff
@@ -0,0 +1,63 @@
+package GET2018.com.metacube.DSA.Assignment4;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Copyright (c) 2018 Metacube.com. All rights reserved.
+ * This class creates collection of employee data structure.
+ * @author Chirag Jain
+ *
+ */
+public class EmployeeCollection {
+
+	private List<Employee> employee = new ArrayList<Employee>();
+	private Set<Integer> employeeId = new HashSet<Integer>();
+	
```

Comment:
Instead of using a ArrayList or HashSet you can simply use a Map<Integer,Employee> employeeMap = new Map<Integer,Employee>.
In this you can check if employee is preset by checking the keys because map.keySet() returns a set. So it will be like add employee if (!employeeMap.containsKey(empId)